docs/naming-conventions.md

```diff
@@ -0,0 +1,42 @@
+# Naming Conventions
+
+All source files and variable names should adhere to the guidelines outlined in this document.
+
+## Rust Code
+
+All variable names, function names, module names, and macros in Rust code should follow typical *snake_case* conventions.
+
+As an additional rule, the functions of all component ffi/src/lib.rs files should also follow *snake_case*, but with an additional prefix based on the library of that function. 
+
+Examples:
+
```

Comment:
Triple backticks for code blocks:

```
foo
bar
baz
```

Reply:
Either indent or use backticks but not both. I prefer backticks for consistency with other markdown, which is why I suggested it.

Mixing indent and backticks makes the code be indented, as whitespace in fenced code blocks is literal.